tests/test_missing.py

```diff
@@ -13,50 +12,59 @@
 import utils
 
 
-def _doit(testfile):
+def _doit(testfile, **kwargs):
     """ 
     Compare and contrast vanilla mean with actual means
     """
+    index = (slice(0, 2), slice(4, 6), slice(7, 9))
     uri = utils.write_to_storage(testfile)
     active = Active(uri, "data", utils.get_storage_type())
     active._version = 0
-    d = active[0:2, 4:6, 7:9]
+    d = active[index]
     mean_result = np.mean(d)
 
-    active = Active(uri, "data", utils.get_storage_type())
+    active = Active(uri, "data", utils.get_storage_type(), **kwargs)
     active._version = 2
     active.method = "mean"
     active.components = True
-    result2 = active[0:2, 4:6, 7:9]
+    result2 = active[index]
     np.testing.assert_array_equal(mean_result, result2["sum"]/result2["n"])
 
+    # Sanity check that the result without missing data info is different.
+    active = Active(uri, "data", utils.get_storage_type())
+    active._version = 2
+    active.method = "mean"
+    active.components = True
+    result3 = active[index]
+    with pytest.raises(AssertionError):
+        np.testing.assert_array_equal(result2["sum"]/result2["n"], result3["sum"]/result3["n"])
```

Comment:
looks to me like you are comparing two identical results but expecting them to be different? These are not run on S3 files but rather, on POSIX netCDFs generated on the fly - look at the regular tests on Push that fail exactly the same. Could you pls explain it to me the point of this PR, maybe we can `_doit` (as Bryan loves the name of that silly func haha) together :grin:

Reply:
hmm, maybe I misunderstood this. I assumed you need to pass in the missing data values to the constructor, but I now see that it will try to determine them from the netcdf file if they have not been provided:

https://github.com/valeriupredoi/PyActiveStorage/blob/main/activestorage/active.py#L99

So perhaps this PR is not required.